src/hooks/useOrgHooks.js

```diff
@@ -1,50 +1,27 @@
 import { useEffect, useRef, useState } from 'react'
-import gql from 'graphql-tag'
-import { GraphQLWrapper } from '@aragon/connect-thegraph'
 
-import { ConvictionVoting } from '@1hive/connect-thegraph-conviction-voting'
+import connectConviction from '@1hive/connect-conviction-voting'
 import { connect } from '@aragon/connect'
-import { getDefaultChain } from '../local-settings'
-import { transformConfigData, getAppAddressByName } from '../lib/data-utils'
-import { addressesEqual } from '../lib/web3-utils'
 import { useContractReadOnly } from './useContract'
-
-import BigNumber from '../lib/bigNumber'
-
-import vaultAbi from '../abi/vault-balance.json'
-import { useWallet } from '../providers/Wallet'
 import {
   useProposalsSubscription,
   useStakesHistorySubscription,
 } from './useSubscriptions'
+import { useWallet } from '../providers/Wallet'
 
-// Endpoints TODO: Move to endpoints file
+// utils
+import { getNetwork } from '../networks'
+import BigNumber from '../lib/bigNumber'
+import { addressesEqual } from '../lib/web3-utils'
+import { getDefaultChain } from '../local-settings'
+import { transformConfigData, getAppAddressByName } from '../lib/data-utils'
 
-// Organzation
-const ORG_ADDRESS = '0xe9869a0bbc8fb8c61b7d81c33fa2ba84871b3b0e'
-const ORG_SUBRAPH_URL =
-  'https://api.thegraph.com/subgraphs/name/1hive/aragon-xdai'
+// abis
+import minimeTokenAbi from '../abi/minimeToken.json'
+import vaultAbi from '../abi/vault-balance.json'
 
 // Convcition voting
-const APP_NAME = 'conviction-voting'
-const APP_SUBRAPH_URL =
-  'https://api.thegraph.com/subgraphs/name/1hive/aragon-conviction-voting-xdai'
-
-// Tokens app
-const TOKEN_MANAGER_SUBGRAPH =
-  'https://api.thegraph.com/subgraphs/name/1hive/aragon-tokens-xdai'
-const TOKEN_HOLDER_QUERY = `
-query miniMeToken($id: String!, $address: String!) {
-  miniMeToken(id: $id){
-    id
-    totalSupply
-    holders(where: {address: $address}){
-      address
-      balance
-    }
-  }
-}
-`
+const APP_NAME = 'conviction-beta' // TODO: save elsewhere
```

Comment:
I was thinking we should co-locate all these configuration related constants either in an `environment.js` file so we can change it with environment variables, or even lift them up into a context so we can change some things at runtime.

Reply:
Yep let's do it now!

Reply:
Went forward with adding env variables for the org address and app name.

Org address defaults to the ones specified at `networks.js` file if none sepcified
App name defaults to `conviction-voting` if none specified.

Reply:
Makes sense! :)